tutorials/getting_started/tips-and-tricks.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: documentation
+title:  Tips and Tricks
+---
+
+# Tips and Tricks
+
+This section is dedicated to generic tips and tricks to make your use and configuration of openHAB easier.
+They are discussed in no particular order.
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+## Backups
+
+Always create and maintain a recent backup of your openHAB config.
+When installed on Linux there will be an `openhab-cli` command line tool that supports backup and restore.
+For other systems, under `$OH_HOME/runtime/bin` you will find `backup.sh` and `restore.sh` (or `.bat`).
+Note that these will back up both text based configs and anything done through the UI.
+However, it only does backup and restore for openHAB, not external services.
+
+When you maintain backups, you have the freedom to explore and experiment without risking breaking everything.
+
+## Don't Wait, Try it Out!
+
+If you know what you want to try but you just don't know what will happen, don't post a new thread on the forum or spend hours reading through the docs and the forum.
+Instead, try it out and see what happens.
+If you have backups, there is little you could do that will actually break things.
+
+## Developer Sidebar
+
+MainUI has a tool called the Developer Sidebar.
+Bring up the sidebar through Developer Tools or the key combination <kbd>Alt-Shift-D</kbd>.
```

Comment:
I think it looks better as three single keys on the page.

```suggestion
Bring up the sidebar through Developer Tools or the key combination <kbd>Alt</kbd><kbd>Shift</kbd><kbd>D</kbd>.
```

Reply:
That looks good to me. I'll accept this change and commit one more update to clear out those pesky MD linting errors I introduced on the last commit and we can call this done. Woohoo!